src/js/utils.js

```diff
@@ -41,3 +41,8 @@ export function decreaseAllMemoIndexes() {
     memo.style.zIndex = --index;
   }
 };
+
+export function getCurrentWorkspace() {
+  const hash = window.location.hash.substr(1).replace(/\//ig, "/");
+  return hash || "manifest_memos";
+};
```

Comment:
This implementation looks great for the most part. Thanks for having a go at it.

However I'd have to agree with @OllysCoding's comment. Relying on the URL explicitly to use this is a bit hidden and hacky (in a good, fun way), but by rendering the feature useless for PWAs (without redownloading) I think is a bit of a deal breaker. 

This is one of these features I had thought about at the beginning but didn't proceed with it myself for I couldn't come up with an interaction model that I was happy with, or that could scale.

Reply:
@this-fifo Always happy to keep #118 open if you'd want to discuss further some alternative models for interaction or introduction workspaces.

Reply:
@jonathontoon I fully agree with you and I also hesitated on this because of the PWA support looking more like a workaround rather than full support but I wanted to bring it up to spark the conversation since it was trivial to add _(thanks to the beautiful dependency free small footprint of the app)_

something that occurred to me was to try following the same approach as the dark theme toggle, basically offer a dedicated workspace switcher UI through a shortcut (similar to alt+tab on the OS for applications, Alt + W maybe?)

that being said, in the spirit of keeping things clean and minimal, I feel it would be important to bring up the point that maybe supporting an infinite amount of workspaces is overkill and manifest could instead allow for up to N workspaces (which would keep the workspace switcher UI cleaner)

I work mostly on a laptop screen and my notes sometimes go pretty lengthy, so screen real estate is definitely a concern for me but I think 5 workspaces would be more than sufficient for most cases I can think off

what do you think?

Reply:
on a totally different tangent, not sure if you want to open an issue for this or not, but on linux (ubuntu + firefox) the Alt + T theme shortcut conflicts with a system shortcut that opens the browser tools menu for me

Reply:
I think a hotkey to bring up some UI for switching workspaces would be an appropriate solution. 

Alternatively, if we're not intent on having unlimited workspaces, that hotkey could be `alt+n` where n is 0-9, giving the user 10 workspaces without any need for changes in the UI. Just a thought!

Reply:
@this-fifo @ollyscoding since we are basically ideating now I'm going to close this and we can move the conversation to #118. 

My final thoughts are: 

One approach could be to have 5 workspaces. Center, right, left, top and bottom. That could be controlled by arrow keys plus some other modifier?

@ollyscoding idea of using numbers could be an interesting one too.

These both might not work on mobile or tablets without a keyboard, but that's a caveat I'd be work willing to let slide. Also regarding the Linux shortcut issue it is being tracked here on #112. Feel free to take on a fix if you'd like @this-fifo.